hw_4/task_3.js

```diff
@@ -0,0 +1,40 @@
+function word(wordSet) {
+    let word = wordSet.toLocaleLowerCase()
+vowels = "aeiou"
+vowels_count = 0
+consonants_count = 0
+for(const i of word){
+    if(vowels.includes(i)) vowels_count++; 
+	else if (typeof i === "symbol") {
+		consonants_count++;
+	}
+}
+console.log(`word contains vowels: ${vowels_count}, and  consonants: ${consonants_count}`);
+}
+
+word("ab21")
+
+//2
+const str = 'aAbZzyZ';    // 'bBcAazA' for +1 and 'zZaYyxY' for -1
```

Comment:
Try to modify this solution to handle such case ab 1Z - leave space and number in the result line